src/cli/remotes-wizard.js

```diff
@@ -8,21 +8,24 @@ export default async function (testCafe, remoteCount, showQRCode) {
 
     if (remoteCount) {
         log.hideSpinner();
+        if (remoteCount === 1)
+            log.write(`Connecting 1 remote browser...\nNavigate to the following URL from the remote browser.`);
```

Comment:
Just use `browser(s)` to avoid this code duplication. No need for `\n`, it's a template string, just use `dedent` and new line (e.g. https://github.com/DevExpress/testcafe/blob/master/src/cli/argument-parser.js#L18)